style.css

```diff
@@ -0,0 +1,53 @@
+body {
+  padding-left: 30px;
+  padding-right: 30px;
+  font-family: 'Poppins', sans-serif;
+  background: url("images/background-image.svg");
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+#menu-icon {
+  width: 20px;
+  height: 20px;
+  background-color: #6070ff;
+}
+
+#logo {
+  color: #6070ff;
+  font-size: 20px;
+  font-weight: bold;
+}
```

Comment:
kindly both of them should be aligned center. it means they should be at the same line. your hamburger is up to your logo. please fix that according to the [Figma templage](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/toolbar-mobile.png) you have choose